model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/stencils/compute_advection_in_vertical_momentum_equation.py

```diff
@@ -196,7 +196,47 @@ def compute_advection_in_vertical_momentum_equation(
     vertical_start: gtx.int32,
     vertical_end: gtx.int32,
 ):
-    """Formerly known as fused_velocity_advection_stencil_15_to_18."""
+    """
+    Formerly known as fused_velocity_advection_stencil_15_to_18.
+
+    This computes the vertical momentum advection in the vertical momentum equation
+
+    Args:
+        - contravariant_corrected_w_at_cells_on_model_levels: contravariant-corrected vertical velocity at model levels
+        - vertical_wind_advective_tendency: vertical advection tendency
```

Comment:
```suggestion
        - vertical_wind_advective_tendency: vertical advection tendency of the vertical wind
```
same as you did for
`        - normal_wind_advective_tendency: horizontal advection tendency of the normal wind`